web/src/util/index.js

```diff
@@ -1,5 +1,7 @@
 /* eslint-disable import/prefer-default-export */
 
+export const ACTIVITY_TYPES = ['HIT', 'HIE', 'MMIS'];
```

Comment:
What do you think about calling this something like `ACTIVITY_FUNDING_SOURCES`?  Or `ACTIVITY_FFP_SOURCE` (Federal funding participation)?  When I see `types` elsewhere in the code, I think about something else altogether.  I'm not sure ***what*** I think, but I consistently have to stop and think about what it means.